src/main/java/org/envirocar/wps/StatsForPOI.java

```diff
@@ -231,10 +234,9 @@ public void simpleAlgorithm(Geometry pointOfInterest, double bufferSize, String
 					int amountOfPoints = 0;
 					
 					String trackID = (String) ((LinkedHashMap)item).get("id");
-					System.out.println("Getting features for track with ID: " + trackID);
 					LOGGER.debug("Getting features for track with ID: " + trackID);
 					URL trackUrl = new URL(EnviroCarWpsConstants.ENV_SERVER_URL+"/tracks/"+trackID+"/measurements"+bboxQuery);
-					System.out.println(" "+ trackUrl);
+					LOGGER.info(" "+ trackUrl);

```

Comment:
TrackURL (URL) was not applicable for .info(String). Will use trackURL.toString() now.